snowshu/core/main.py

```diff
@@ -182,3 +186,86 @@ def launch_docker_cmd(replica: str):
     """Return the docker command line string to start a given replica."""
     replica_manager = ReplicaManager()
     click.echo(replica_manager.launch_docker_command(replica))
+
+@cli.group()
+@click.option(
+    "--type",
+    "-t",
+    help="The type of adapter to use for the operation.",
+    type=click.Choice(["snowflake", "postgres"]),
+    required=True,
+)
+@click.pass_context
+def adapter(ctx, type):
+    """Adapter related commands."""
+    ctx.ensure_object(dict)
+    ctx.obj["TYPE"] = type
+
+
+@adapter.command()
+@click.option(
+    "--credentials-file",
+    default="./replicas/credentials.yml",
+    type=click.Path(exists=True),
+    help="The path to the credentials file.",
+    required=True,
+)
+@click.option(
+    "--prod-prefix",
+    default="SNOWSHU_PROD",
```

Comment:
Should we specify possible values in advance to prevent scenarios where an incorrect database is provided?

Reply:
didn't think about it, will add choice